gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,496 @@
+= Getting Started with Flex Gateway on a Kubernetes Cluster
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:version-helm: 3.0.0
+
+Get started with Anypoint Flex Gateway on a Kubernetes cluster. Set up a Flex Gateway container that connects to Anypoint Platform in Connected Mode, and use the container to control local traffic to an API implementation. This guide covers the following high-level tasks:
+
+* Installing, registering, deploying, and running a Flex Gateway container in Connected Mode. 
+* Creating an API in Anypoint API Manager that runs on your Flex Gateway container. 
+* Connecting to the API, first without any authentication, and then with a basic authentication policy that you set up for the API in API Manager. 
+
+The Flex Gateway deployment process requires a working Kubernetes cluster that you create or that is provisioned for you by a Kubernetes professional. 
+
+Setting up a Kubernetes cluster requires a level of skill and expertise that is _beyond the scope of this guide to provide_. Cluster configuration in Kubernetes differs markedly by service provider and platform. For basic cluster setup questions, consult documentation from your Kubernetes service provider. 
```

Comment:
Third-party setup instructions get outdated and go beyond the scope of normal docs. Instructions for setting up a K8 cluster are complex and require some expertise.

Unless we want to provide a very SIMPLE set of instructions, perhaps on a tool like minikube, there really isn't a way to provide setup steps.

An alternative is to explain the basic requirements of the cluster based on HOW we are/will be using Flex in it. I assume Flex can play different roles (ingress, ingress controller, maybe even egress in some cases?). We need to provide more detail on that so that it fits with the architectural info we're giving people.

Reply:
We are also adding some testing steps to confirm that an already-created cluster is ready for the deployment step.

Find "Check Your Cluster," below.

Reply:
A benefit to showing a SIMPLE cluster setup would be to discover any cluster requirement (gaps) that we should cover in this doc so that we can doc that and provide a way for people to check/test that their cluster meets that requirement.